documentation/angular-ci.asciidoc

```diff
@@ -154,9 +154,9 @@ This will be used as follows:
 +
 [source, groovy]
 ----
-sh """yarn ${buildScript}"""
+`sh """yarn ${buildScript}"""`
```

Comment:
no need of backticks here... this should be ignored by regex from .spellcheck.yml